src/components/Contacts/ContactDetails/ContactDetailsTab/EditContactModal/EditContactModal.tsx

```diff
@@ -0,0 +1,696 @@
+            </Grid>
+            <Grid item xs={6}>
+              <FormControl fullWidth>
+                <InputLabel id="social-type-label">{t('Type')}</InputLabel>
+                <Select labelId="socail-type-label" value={null} fullWidth>
+                  <MenuItem value={'facebook'}>{t('Facebook')}</MenuItem>
+                  <MenuItem value={'twitterk'}>{t('Twitter')}</MenuItem>
+                  <MenuItem value={'linkedin'}>{t('LinkedIn')}</MenuItem>
+                  <MenuItem value={'website'}>{t('Website')}</MenuItem>
+                </Select>
+              </FormControl>
+              <ContactEditDeleteIconButton>
+                <DeleteIcon />
+              </ContactEditDeleteIconButton>
+            </Grid>
+          </Grid>
+        </ContactInputWrapper>
+      </>
+    );
+  };
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      title={'Edit People'}
+      content={
+        <>
+          <ContactEditContainer>
+            <ContactInputWrapper>
+              <ContactInputField
+                label={t('Contact')}
+                value={contact.name}
+                fullWidth
+              />
+            </ContactInputWrapper>
+            <ContactInputWrapper>
+              <PrimaryContactIcon />
+
+              <FormControl fullWidth={true}>
+                <ContactPrimaryPersonSelectLabel id="primary-person-select-label">
+                  {t('Primary')}
+                </ContactPrimaryPersonSelectLabel>
+                <Select
+                  labelId="primary-person-select-label"
+                  value={contact.primaryPerson?.id}
+                  fullWidth={true}
+                >
+                  {contact.people.nodes.map((person) => {
+                    return (
+                      <MenuItem
+                        key={person.id}
+                        value={person.id}
+                      >{`${person.firstName} ${person.lastName}`}</MenuItem>
+                    );
+                  })}
+                </Select>
+              </FormControl>
+            </ContactInputWrapper>
+          </ContactEditContainer>
+          <Divider />
+          <ContactEditContainer>
+            {contact.people.nodes.map((person) => {
+              return (
+                <>
+                  <ContactPersonContaner>
+                    <ContactInputWrapper>
+                      <ContactAvatar
+                        alt={`${person.firstName} ${person.lastName}`}
+                        src={person.lastName ?? ''}
+                      />
+                      <ContactPersonNameText>{`${person.firstName} ${person.lastName}`}</ContactPersonNameText>
+                      <ContactEditDeleteIconButton>
+                        <DeleteIcon />
+                      </ContactEditDeleteIconButton>
+                    </ContactInputWrapper>
+                    <ContactInputWrapper>
+                      <Grid container spacing={3}>
```

Comment:
This could be a contact grid component maybe? Since each person gets the same fields right?